infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServerNode.java

```diff
@@ -67,32 +66,37 @@
      * @param serverContext Initialized Server Context.
      */
     public CorfuServerNode(@Nonnull ServerContext serverContext) {
-        this(serverContext,
-                ImmutableMap.<Class, AbstractServer>builder()
-                        .put(BaseServer.class, new BaseServer(serverContext))
-                        .put(SequencerServer.class, new SequencerServer(serverContext))
-                        .put(LayoutServer.class, new LayoutServer(serverContext))
-                        .put(LogUnitServer.class, new LogUnitServer(serverContext))
-                        .put(ManagementServer.class, new ManagementServer(serverContext))
-                        .build()
-        );
+
+        this(serverContext, createServerMap(serverContext));
```

Comment:
I don't think this "clean up" is enough, or necessary at this point. If you don't need this change for your PR, then I would suggest you revert this change and do a proper clean up in a subsequent PR.

`CorfuServer` , `CorfuServerNode` and `SequencerServer` etc necessary and confusing. We should have one server type and multiple handlers. 

I know you didn't create this but these code re-orgs are really not improving the quality. We need to simplify and reduce the types (i.e. have one server and multiple handlers only).